Source/src/core/GameObject.cpp

```diff
@@ -67,6 +71,7 @@ Hachiko::GameObject::~GameObject()
     {
         RELEASE(component);
     }
+    components_to_remove.clear();
```

Comment:
Is this necessary if the go is going to be deleted?

Reply:
you're right, we don't care at this point